### PR TITLE
Add maven wrapper

### PR DESCRIPTION
**Pull request by eddumelendez**
Maven wrapper helps to build the project in case maven is not installed in the local machine. Also provides the right maven version, which project is working on and avoid compatibility issues with plugins.

### CODE REVIEW

**Comment by heiko-braun** [comment]
@eddumelendez Can you provide a brief description of the purpose, benefit or use case for this enhancement? This would be helpful to understand the context in which the wrapper is going to be used.

File: .mvn/wrapper/maven-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.3.1/apache-maven-3.3.1-bin.zip
```
Comment (emag) [inline .mvn/wrapper/maven-wrapper.properties]: 3.3.9(latest) would be good?

Comment (heiko-braun) [inline .mvn/wrapper/maven-wrapper.properties]: +1

**Comment by eddumelendez** [comment]
@heiko-braun I have added a description and also updated the version to 3.3.9.

**Comment by heiko-braun** [comment]
thanks @eddumelendez

**Pull request merged**